spec/application_controller_spec.rb

```diff
@@ -89,9 +89,8 @@ def app
     expect(body[:message]).to include('Welcome to the Yelp Microservice!')
   end
 
-  # Test is causing issues with Travis - further research necessary
   xit 'can hit the utilities/electricity endpoint' do
-    location = 80_211
+    location = 80211
```

Comment:
Will these tests be unskipped eventually or are we keeping them this way?

Reply:
We are keeping them this way until we figure out what is causing Travis to fail